games-board/chessx/chessx-1.4.6.recipe

```diff
@@ -3,9 +3,18 @@ DESCRIPTION="With ChessX you can operate your collection of chess games \
 in many ways: browse, edit, add, organize, analyze, etc."
 HOMEPAGE="https://chessx.sourceforge.net/"
 COPYRIGHT="
-	2005-2006 Michal Rudolf
+	1998-2005 Gilles Vollant
+	1999-2004 Shane Hudson
+	2005 Ejner Borgbjerg
+	2005-2007 Marius Roets
+	2005-2007 Heinz R. Hopfgartner
+	2005-2010 Michal Rudolf
+	2005-2011 Sergey A. Tachenov
 	2007 Rico Zenklusen
-	2007-2009 Michal Rudolf
+	2008 Aliaksandr Antonik
+	2010 P. Sereno
+	2011 Nokia Corporation and/or it's subsidiary(-ies)
+	2012-2016 Jens Nissen	
 	"
```

Comment:
```diff
-COPYRIGHT="
-	2005-2006 Michal Rudolf
+COPYRIGHT="2005-2006 Michal Rudolf
 	1998-2005 Gilles Vollan
```
```diff
 	2011 Nokia Corporation and/or it's subsidiary(-ies)
-	2012-2016 Jens Nissen	
- 	"
+	2012-2016 Jens Nissen"
 LICENSE="GNU GPL v2"
```
Can you please make the changes above, in order to use the recommended style for `COPYRIGHT`?
Source: https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes#a-simple-example-using-the-recommended-style-for-a-multi-line-copyright
Thanks!